docs_user/modules/openstack-edpm_adoption.adoc

```diff
@@ -622,15 +622,14 @@ oc exec -it nova-cell1-conductor-0 -- nova-manage db online_data_migrations
 * Verify if Nova services can stop the existing test VM instance:
 +
 ----
-${BASH_ALIASES[openstack]} server list | grep -qF '| test | ACTIVE |' && openstack server stop test
-${BASH_ALIASES[openstack]} server list | grep -qF '| test | SHUTOFF |'
-${BASH_ALIASES[openstack]} server --os-compute-api-version 2.48 show --diagnostics test | grep "it is in power state shutdown" || echo PASS
+${BASH_ALIASES[openstack]} server list | grep -qF '| test | ACTIVE |' && ${BASH_ALIASES[openstack]} server stop test || echo PASS
+${BASH_ALIASES[openstack]} server list | grep -qF '| test | SHUTOFF |' || echo FAIL
+${BASH_ALIASES[openstack]} server --os-compute-api-version 2.48 show --diagnostics test 2>&1 || echo PASS
 ----
 
 * Verify if Nova services can start the existing test VM instance:
 +
 ----
-${BASH_ALIASES[openstack]} server list | grep -qF '| test | SHUTOFF |' && openstack server start test
-${BASH_ALIASES[openstack]} server list | grep -F '| test | ACTIVE |'
-${BASH_ALIASES[openstack]} server --os-compute-api-version 2.48 show --diagnostics test --fit-width -f json | jq -r '.state' | grep running
-----
```

Comment:
Ah the breakage probably wasn't in the Ceph docs, it's here

Reply:
what is the breakage here?

Reply:
nvm, got it :) got to add ``----`` back, right?